examples/prometheus/prometheus.yaml

```diff
@@ -16,8 +16,17 @@ parameters:
   name: IMAGE_PROXY
   value: openshift/oauth-proxy:v1.0.0
 - description: The location of the prometheus image
+  # TODO: Change to official openshift build
   name: IMAGE_PROMETHEUS
-  value: openshift/prometheus:v2.0.0-dev
```

Comment:
Because prometheus 2.0 doesn't support flags that only have a single dash.  Please revert this.